install/kubernetes/cilium/charts/operator/templates/clusterrole.yaml

```diff
@@ -53,3 +53,9 @@ rules:
   - ciliumidentities/status
   verbs:
   - '*'
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
```

Comment:
Based on the suggestions we would need to add `list` and `watch` verbs here.

Reply:
Done